spack_repo/packages/mmm/package.py

```diff
@@ -15,7 +15,7 @@ class Mmm(CMakePackage):
     url      = "https://github.com/thelfer/mfem-mgis.git"
 
     version('develop', git='https://github.com/thelfer/mfem-mgis.git',
-        branch='master', submodules='True')
+        branch='latug0_spack_cmake', submodules='True')
```

Comment:
Temporary ?

Reply:
Yes, testing purpose. It was required in order to test thse command "spack install mmm".
The modification is removed now. We are back to "master" branch.